### ISSUE TITLE
Setup Github Actions

### ISSUE THREAD

**Issue by psteininger** [vt-elixir/ja_resource#83]
Remove Travis CI setup

**Comment by ericdude4** [comment]
My PR was only failing the Travis CI setup. https://github.com/vt-elixir/ja_resource/pull/82

**Comment by psteininger** [comment]
that's right @ericdude4. Speaking of the setup, would you have suggestions for which Elixir and OTP versions we should target? With Github actions it's really easy to set up a matrix of versions for both. At one end I think it would be Elixir 1.9 and OTP 22, but how far back should we go?

**Comment by ericdude4** [comment]
@psteininger I responded over on https://github.com/vt-elixir/ja_resource/pull/82

> Since I assume we are going to set the major version to 1.x, we can make the dependencies as up-to-date as we want. I would say we follow suit of the latest Phoenix 1.4 dependencies (provided that it doesn't break anything) since that will be the environment that most new projects will already have.

> Currently, Phoenix 1.4 supports Elixir 1.5 and OTP 19. The Phoenix 1.5.0-dev build supports Elixir 1.6, which itself requires OTP 19. I would say we should be compatible with the most recent stable release of Phoenix. Elixir ~> 1.7 is a higher requirement than even the development build of Phoenix. Also, Elixir 1.10 just released :) https://elixir-lang.org/blog/2020/01/27/elixir-v1-10-0-released/

> The requirements for Phoenix 1.4 that I think we should match in this repository are:

> Elixir ~>1.5
> OTP 19
> Ecto ~>3.0

**Comment by psteininger** [comment]
@ericdude4 I had a pretty busy couple of weeks, and it looks like some more hustle is ahead. I hope I will have more time in a couple of weeks and can crank out some updates and get a new version pushed out.
